libraries/chain/asset_evaluator.cpp

```diff
@@ -656,7 +656,7 @@ static bool update_bitasset_object_options(
       const asset_update_bitasset_operation& op, database& db,
       asset_bitasset_data_object& bdo, const asset_object& asset_to_update )
 {
-   const fc::time_point_sec& next_maint_time = db.get_dynamic_global_properties().next_maintenance_time;
+   const fc::time_point_sec next_maint_time = db.get_dynamic_global_properties().next_maintenance_time;
```

Comment:
I think `next_maintenance_time` is a tiny object so it's better to pass by value rather than by reference (like everywhere else). Since I was modifying this function, I modified this.

Reply:
I haven't found a clear argument for or against either in the specific case of small objects. Ok.